roles/kubernetes/preinstall/tasks/0120-growpart-azure-centos-7.yml

```
@@ -7,6 +7,10 @@
     name: cloud-utils-growpart
     state: present
 
+- name: Gather host facts to get ansible_mounts
+  setup:
+    filter: ansible_mounts
+
```

I suggest using a `gather_subset` rather than filtering the already gathered facts:
```suggestion
- name: Gather mounts facts
  setup:
    gather_subset: 'mounts'
```

---

@ralbon Are you available to address this comment ? I would love to include this one in the 2.18 release but I have to set a deadline

---

Thank you, Done